SEOImageUpdates: User Story 1463880, Small PR 2 (#12338)
dotnet/docs: docs/framework/windows-workflow-foundation/wf-features-in-the-rehosted-workflow-designer.md

 The following screenshot shows the variable and argument designer context menu.

 ![Variable and Argument Designer Context Menu](./media/wf-features-in-the-rehosted-workflow-designer/designer-context-menu.png)

### Auto-surround with Sequence
 Since a workflow or certain container activities (such as <xref:System.Activities.Statements.NoPersistScope>) can only contain a single body activity, adding a second activity required the developer to delete the first activity, add a <xref:System.Activities.Statements.Sequence> activity, and then add both activities to the sequence activity. Starting with [!INCLUDE[net_v45](../../../includes/net-v45-md.md)], when adding a second activity to the designer surface, a `Sequence` activity will be automatically created to wrap both activities. This feature is supported in the rehosted designer.

 The following screenshot shows a `WriteLine` activity in the `Body` of a `NoPersistScope`.

>  ![Auto-surround with WriteLine activity](./media/wf-features-in-the-rehosted-workflow-designer/auto-surround-write-line-activity.png)

Reviewer: ```suggestion
 ![A WriteLine activity in the Body of a NoPersistScope activity.](./media/wf-features-in-the-rehosted-workflow-designer/auto-surround-write-line-activity.png)
```

 The following screenshot shows the automatically created `Sequence` activity in the `Body` when a second `WriteLine` is dropped below the first.

>  ![Auto-surround with Sequence activity](./media/wf-features-in-the-rehosted-workflow-designer/auto-surround-sequence-activity.png)

Reviewer: ```suggestion
 ![An automatically created Sequence in the Body of a NoPersistScope.](./media/wf-features-in-the-rehosted-workflow-designer/auto-surround-sequence-activity.png)
```

### Pan Mode
 To more easily navigate a large workflow in the designer, pan mode can be enabled, allowing the developer to click and drag to move the visible portion of the workflow, rather than needing to use the scroll bars. The button to activate pan mode is in the lower right corner of the designer. This feature is supported in the rehosted designer.

 The following screenshot shows the pan button which is located at the bottom right corner of the workflow designer.

>  ![Use the pan button to move the visible portion of the workflow.](./media/wf-features-in-the-rehosted-workflow-designer/pan-button-visible-workflow.png)

Reviewer: ```suggestion
 ![The pan button highlighted in the workflow designer.](./media/wf-features-in-the-rehosted-workflow-designer/pan-button-workflow-designer.png)
```

 The following screenshot of the completed workflow from the [Getting Started Tutorial](getting-started-tutorial.md) shows the outline view with a sequential workflow.

>  ![Outline View with a sequential workflow](./media/wf-features-in-the-rehosted-workflow-designer/outline-view-in-workflow-designer.jpg)

Reviewer: ```suggestion
 ![Screenshot of outline view with a sequential workflow in Visual Studio](./media/wf-features-in-the-rehosted-workflow-designer/outline-view-in-workflow-designer.jpg)
```

  
 The following screenshot shows the completed state machine workflow from the [Getting Started Tutorial](getting-started-tutorial.md) step [How to: Create a State Machine Workflow](how-to-create-a-state-machine-workflow.md).  
  

>  ![Completed state machine workflow from the Getting Started Tutorial](./media/wf-features-in-the-rehosted-workflow-designer/getting-started-tutorial-complete-state-machine-workflow.jpg)  

Reviewer: ```suggestion
 ![Illustration that shows the completed state machine workflow.](./media/wf-features-in-the-rehosted-workflow-designer/complete-state-machine-workflow.jpg)  
```
